src/tool_getparam.c

```diff
@@ -581,8 +582,7 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
 
   if(('-' != flag[0]) || ('-' == flag[1])) {
     /* this should be a long name */
-    const char *word = ('-' == flag[0]) ? flag + 2 : flag;
-    size_t fnam = strlen(word);
+    const char *word = flag + 2;
```

Comment:
What if `flag` doesn't start with two dashes? Just removing that condition surely will change the logic here?